ember-cli-build.js

```diff
@@ -11,6 +11,9 @@ module.exports = function(defaults) {
       config: {
         excludes: ['utils/.*']
       }
+    },
+    'ember-cli-babel': {
+      includePolyfill: true
```

Comment:
This was for regenerator/async-await style tests, `Promise` doesn't exist in PhantomJS yet (trolled me so hard 😛)